contrib/experimental/great_expectations_experimental/expectations/expect_column_values_are_probabilistically_greater_than_equal_to_threshold.py

```diff
@@ -0,0 +1,170 @@
+                    "McRoomyRoom",
+                ],
+                "INSTURL": [
+                    "www.aamu.edu/",
+                    "https://www.uab.edu",
+                    "www.amridgeuniversity.edu",
+                    "McRoomyRoom",
+                ],
+                "NPCURL": [
+                    "www.aamu.edu/admissions-aid/tuition-fees/net-price-calculator.html",
+                    "https://uab.studentaidcalculator.com/survey.aspx",
+                    "www2.amridgeuniversity.edu:9091/",
+                    "McRoomyRoom",
+                ],
+                "LATITUDE": ["34.783368", "33.505697", "32.362609", "McRoomyRoom"],
+                "LONGITUDE": ["-86.568502", "-86.799345", "-86.17401", "McRoomyRoom"],
+                "RELAFFIL": ["NULL", "NULL", "74", "McRoomyRoom"],
+                "DEATH_YR2_RT": [
+                    "PrivacySuppressed",
+                    "PrivacySuppressed",
+                    "PrivacySuppressed",
+                    "McRoomyRoom",
+                ],
+                "SEARCH_STRING": [
+                    "Alabama A & M University AAMU",
+                    "University of Alabama at Birmingham ",
+                    "Amridge University Southern Christian University  Regions University",
+                    "McRoomyRoom",
+                ],
+            },
+            "tests": [
+                {
+                    "title": "positive_test_with_column_one",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {"column": "OPEID6", "threshold": 0.00},
+                    "out": {
+                        "success": True,
+                        "unexpected_index_list": [],
+                        "unexpected_list": [],
+                    },
+                },
+                {
+                    "title": "error_test_threshold",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {"column": "ZIP", "threshold": 0.61},
+                    "out": {
+                        "success": False,
+                    },
+                },
+            ],
+        }
+    ]
+
+    # This is the id string of the Metric used by this Expectation.
+    # For most Expectations, it will be the same as the `condition_metric_name` defined in your Metric class above.
+    map_metric = (
+        "column_values.prediction_confidence_greater_than_or_equal_to_threshold"
+    )
+
+    # This is a list of parameter names that can affect whether the Expectation evaluates to True or False
+    success_keys = (
+        "threshold",
+        "mostly",
+    )
+
+    # This dictionary contains default values for any parameters that should have default values
+    default_kwarg_values = {}
+
+    # This object contains metadata for display in the public Gallery
+    library_metadata = {
```

Comment:
@taylorfturner - could you please add a `requirements` key to the the `library_metadata` object with a list of any additional dependencies? It looks like in addition to `dataprofiler`, `tensorflow` and `scikit-learn` are requirements. [You can see an example here.](https://github.com/great-expectations/great_expectations/blob/develop/contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_us_zipcode_within_radius_of_given_zipcode.py#L245). You'll also see that it is included in the docstring on L38.

Reply:
got it